Logic.json

```diff
@@ -52,12 +52,12 @@
 
 	"Intermediates": {
 		"Explosives": [["Grenade_Eggs"], ["Clockworks"]],
-		"Clockwork_Warp": [["Grenade_Eggs", "Egg_Aim", "Clockworks"]],
+		"Clockwork_Warp": [["Grenade_Eggs", "Egg_Aim", "Clockworks", "Glitches"]],
 		"MT_Flight": [["Bill_Drill"], ["MT_Mumbo"]],
 		"Relic": [["Egg_Aim", "Grip_Grab"], ["MT_Flight", "Clockwork_Warp"]],
-		"JSG": [["MT_Flight"], ["MT_Mumbo"]],
-		"Code_Chamber": [["JSG", "MT_Mumbo", "Egg_Aim"], ["JSG", "Explosives", "Egg_Aim"]],
-		"Prison_Compound": [["MT_Mumbo"], ["Explosives"]],
+		"JSG": [["MT_Flight", "Glitches"], ["MT_Mumbo"]],
+		"Code_Chamber": [["MT_Mumbo", "Egg_Aim"], ["JSG", "Explosives", "Egg_Aim"]],
+		"Prison_Compound": [["MT_Mumbo"], ["Explosives"], ["MT_Flight", "Glitches"]],

```

Comment:
I'd say insane instead of glitches

Reply:
If you do vine clip instead of doing the dumb clip through the door, I wouldn't say it'd be that hard

Reply:
ah ok 👍